src/ovp_pipeline/digest_config.py

```
@@ -0,0 +1,277 @@
+  :mod:`digest_inputs`.
+* Action mutations — Stage 4 / Stage 5 of the M23 plan add their own
+  audit-event emission helpers; this module is read-only.
+"""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass, field
+from pathlib import Path
+from types import MappingProxyType
+from typing import Any, Final, Mapping
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)